modules/apps/transfer/keeper/migrations.go

```diff
@@ -0,0 +1,52 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/cosmos/ibc-go/v4/modules/apps/transfer/types"
+)
+
+// Migrator is a struct for handling in-place store migrations.
+type Migrator struct {
+	keeper Keeper
+}
+
+// NewMigrator returns a new Migrator.
+func NewMigrator(keeper Keeper) Migrator {
```

Comment:
is the intention that all transfer related migrations will happen here?

Reply:
yes -- the cosmos sdk has all migrations in a [separate directory](https://github.com/cosmos/cosmos-sdk/tree/main/x/bank/migrations) but imo that might be overkill for us. wdyt?

Reply:
I think for us it makes sense to have them here, we can worry about additional directories if we end up with a large number of migrations.